modules/issue_tracker/test/issue_tracker_test_plan.md

```diff
@@ -1,37 +1,36 @@
-Issue Tracker Filter Form [Automation Testing]
-1. A user can access the module only if they have one or more of the `issue_tracker_reporter` or `issue_tracker_developer` permissions.
+# Issue Tracker Test Plan
+
+## Issue Tracker Filter Form [Automation Testing]
+1. User can access the page iff they have issue tracker reporter or developer permission
```

Comment:
There's a trend away from using "iff" in our test plans since it's more work explaining it to people who don't know the word than just writing out the whole thing.